.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "editor.insertSpaces": true,
+  "editor.tabSize": 2,
+  "editor.detectIndentation": false,
+  "files.eol": "\n",
+  "files.insertFinalNewline": true
+}
```

Comment:
How about adding something like this:
```json
// Set the default
"editor.formatOnSave": false,
```

Or maybe doing it for a specific language:
```json
"[javascript]": {
    "editor.formatOnSave": true
}
```

More [here](https://github.com/prettier/prettier-vscode)

Reply:
Yeah, that might be nice, so it happens while you're in the tool without having to run `eslint -fix` or prettier

Reply:
Yea, this is a good idea. I will have that in.
I was wondering between leaving the ability of free-styling to the developer until they make a commit (we already have a precommit hook) or enforcing it every time they save a file, too.